### ISSUE TITLE
arm64 images don't exist so you can't use with arm64 clusters

### ISSUE THREAD

**Issue by bc-sb** [Azure/Bridge-To-Kubernetes#273]
**Describe the bug**
There are instructions on how to build multiplatform agents, but the default images for k8s don't appear to be multiplatform (bridgetokubernetes.azurecr.io/lpkrestorationjob:1.3.0 and bridgetokubernetes.azurecr.io/lpkremoteagent:1.3.0)

**Mention the platform you are using**
I am using a wsl2-ubuntu 20.04 enviro attempting to connect to an arm64 eks cluster.  Using a golang service.

**To Reproduce**
1. Spin up an arm64 based cluster
2. Pick a default golang app
3. Run vscode and install bridge to kubernetes
4. Configure cluster / app to connect
5. Attempt to connect
6. stern logs for container
7. see messages like exec /usr/bin/dotnet: exec format error and exec /app/restorationjob: exec format error

**Expected behavior**
That error is a telltale sign of a arch mismatch.  When I inspect the docker image I don't see a arm64 section (docker buildx imagetools inspect bridgetokubernetes.azurecr.io/lpkremoteagent:1.3.0 | grep arm64)

**Screenshots**

**Desktop (please complete the following information):**
- Windows 11 w/ wsl2 running Ubuntu 20.04
- 
**Smartphone (please complete the following information):**

**Additional context**
I pulled this repo and noticed that there are some enviro vars that look to be available if building from scratch, but I don't see a way to override it on a existing build.  I built both the agent and job using buildx multiplatform, but I can't seem to get the bridge-to-kubernetes.resource to honor my overrides using something like: MINDARO_DEVHOSTAGENT_RESTORATIONJOB_TAG or  BRIDGE_DEVHOSTIMAGENAME

**Comment by hsubramanianaks** [comment]
@bc-sb Thanks for raising this issue. Official builds don't have **arm 64** binaries for B2K yet. One way you can achieve this is by building the project yourself and point to those binaries. For example:
check out this repo and run the following ```dotnet publish -r linux-arm64 -c Release --self-contained false``` and then set the environment variable **before launching your vscode instance**. 
```
export BRIDGE_BUILD_PATH=/home/<yourusername>/repos/.../src/dsc/bin/Release/net6.0/linux-arm64/publish 
export BRIDGE_RESTORATIONJOBIMAGENAME=yourimageurl 
export BRIDGE_ROUTINGMANAGERIMAGENAME=yourimageurl 
export BRIDGE_DEVHOSTIMAGENAME=yourimageurl

``` 

to build the images below are the commands.

```
docker build -f ./src/routingmanager/Dockerfile -t <your registry>/<your imagename, e.g. michelleroutingmanager> . 
```

push them to your registry or public one like docker.io
```docker push <your registry>/<your imagename>```

once all these environment variables are set, launch vscode and B2K vscode extension would refer to these variables. 

this is quite lot of steps, until we make this arm 64 binaries available via official builds. These steps are the workaround to make it work. Please let me know if you see any issues. Thank you for your patience.

**Comment by hsubramanianaks** [comment]
@bc-sb did the above instructions to generate arm 64 images work for you ? Please let us know. Also are you still looking for arm 64 images for using b2k still ? Thank you.

**Comment by bc-sb** [comment]
I was able to get it working with a bunch of fiddling as you outlined -- I
forget if there was any other dials I had to turn at the time.  I would
love a docker image that supported multiple platforms as it appears to be
becoming more and more prevalent, so it would be great if it 'just worked'.
We had to add mult-platform support for our images end of last year as we
are seeing arm64 becoming more and more mainstream (gravitron, apple,
etc).  One of the reasons the Bridge-To-Kubernetes caught my eye was the
initial ease of use that allowed a developer with not much k8s
experience to be able to hit the ground running on remote containers with
powerful vscode debug tools. This would be ideal as we have to support both
platforms for devving.
